data/newsletter/issue-382.markdown

```diff
@@ -0,0 +1,60 @@
+Welcome to another issue of Haskell Weekly!
+[Haskell](https://www.haskell.org) is a safe, purely functional programming language with a fast, concurrent runtime.
+This is a weekly summary of what's going on in its community.
+
+## Featured
+
+- [GHC 9.8.1-alpha3 is now available](https://discourse.haskell.org/t/ghc-9-8-1-alpha3-is-now-available/7399) by Ben Gamari
+> The GHC developers are very pleased to announce the availability of the third alpha prerelease of GHC 9.8.1.
```

Comment:
Just so you know, these need to be indented at least two spaces. Like this:

``` md
- [Example Title](http://url.example) by Example Author
  > Example blurb.
```

Otherwise it will generate HTML like this:

``` html
<ul>
  <li>
    <a href="http://url.example">Example Title</a>
    by Example Author
  </li>
</ul>
<blockquote>
  Example blurb.
</blockquote>
```

Reply:
Oh woops, thank you!